kolibri/plugins/learn/assets/src/views/LibraryPage/OtherLibraries.vue

```diff
@@ -55,16 +51,25 @@
               />
             </span>
           </span>
-          <span
+          <div
             v-show="!searchingOtherLibraries && !devicesWithChannelsExist"
-            data-test="no-other"
+            class="a"
```

Comment:
CSS class names are most effective when they are descriptive and meaningful, and indicate the purpose of the element. It would be best to avoid using vague or generic names. 

Maybe instead of class names `a` and `b`, something like `no-other-div` and `no-other-label-div`? This will also make the CSS easier to read and helps other devs quickly identify which element it is referring to.